function/build.gradle

```diff
@@ -0,0 +1,19 @@
+plugins {
+    id 'java-library'
+    id 'io.deephaven.project.register'
+}
+
+dependencies {
+    api project(':qst')
```

Comment:
Do we really need to depend on QST here? We can't be "pure Java"?

Reply:
I'll merge and fix after https://github.com/deephaven/deephaven-core/pull/5054 lands